### PR TITLE
Conduct

### PR DESCRIPTION
**Pull request by RickWinter**
Add a code of conduct file.

### CODE REVIEW

**Review by antkmsft — COMMENTED** [review]
It looks like there's way more than a single code of conduct file in this PR.

**Comment by ahsonkhan** [comment]
> Add a code of conduct file.

What's the goal of this PR?

The title and description don't match the set of files changed, which include storage blob APIs.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```
Comment (ahsonkhan) [inline CODE_OF_CONDUCT.md:1]: Do we plan to do this across all the language SDKs? Should we copy them verbatim?
https://github.com/Azure/azure-sdk-for-java/blob/master/CODE_OF_CONDUCT.md
https://github.com/Azure/azure-sdk-for-python/blob/master/CODE_OF_CONDUCT.md

.NET doesn't seem to have it as a separate doc, but rather part of the contributing section:
https://github.com/Azure/azure-sdk-for-net#contributing

Comment (RickWinter) [reply CODE_OF_CONDUCT.md:1]: I took the Code of Conduct that OSS requires us to have in the repo, https://github.com/microsoft/repo-templates/blob/main/shared/CODE_OF_CONDUCT.md

**Pull request closed without merging**